scripts/assets.js

```diff
@@ -0,0 +1,129 @@
+import { provider as UI, Image } from '@dropins/tools/components.js';
+import { getConfigValue } from './configs.js';
+
+/** Reports whether AEM Assets are being used as the asset source. */
+export async function isAemAssetsEnabled() {
+  const config = await getConfigValue('commerce-assets-enabled');
+  return config === 'true';
+}
```

Comment:
FYI this will be changing soon and will no longer require `await` :)

https://github.com/hlxsites/aem-boilerplate-commerce/pull/387/files#diff-eb55b6bcdfbd2110ceed72aa446694d87208d331fbcf59d2887701c142611ccfR201